packages/analytics-js-integrations/src/integrations/GoogleAds/browser.js

```diff
@@ -81,14 +84,8 @@ class GoogleAds {
     return this.isLoaded();
   }
 
```

Comment:
same here, is there new way of handling in core sdk for unsupported message types ?

Reply:
it's already handling. that's why we removed it